packages/celotool/src/lib/oracle.ts

```diff
@@ -422,11 +434,16 @@ function rbacReleaseName(celoEnv: string) {
   return `${celoEnv}-oracle-rbac`
 }
 
-async function rbacAuthTokenName(celoEnv: string) {
+async function rbacAuthTokenName(celoEnv: string, replicas: number) {
+  const names = [...Array(replicas).keys()].map(i => `${rbacReleaseName(celoEnv)}-${i}`)
   const [tokenName] = await execCmdWithExitOnFailure(
-    `kubectl get serviceaccount --namespace=${celoEnv} ${rbacReleaseName(
-      celoEnv
-    )} -o=jsonpath="{.secrets[0]['name']}"`
+    `kubectl get serviceaccount --namespace=${celoEnv} ${names.join(' ')} -o=jsonpath="{.items[*].secrets[0]['name']}"`
   )
-  return tokenName.trim()
+  const tokenNames = tokenName.trim().split(" ")
```

Comment:
nit: Generally in typescript/javascript we tend to use single quotes for strings